### ISSUE TITLE
[A11y 8/25] Keyboard navigation 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-dropdown#23]
I'm not able to see focus state when using just the keyboard (no screen reader). I should be able to see which list item I am on in each drop down by using the arrow keys

On Chrome OS with ChromeVox, the experience is a bit confusing. When I open a dropdown, I should be able to use the arrow keys to go up and down it and hear the items read aloud. This does not happen. When I use Chromevox navigation (shift + search + down arrow) I can't access the items. I can only access them if I happen to navigate backwards  after opening the list -- i.e. shift + search + up arrow. This is not intuitive. I should be able to open the dropdown with keyboard, use the arrows alone to navigate through, or if I choose, use the screen reader navigation keys to navigate through in logical order.

**Comment by lpalmaro** [comment]
Also, please note that on Windows, I can't press enter to activate one of the list items (and thus close the dropdown menu). Not sure if I'm supposed to be able to do this.

**Comment by blasten** [comment]
This issue is fixed.
